spooky-oysters/bangazon
# Dr orders with products

## Link to Ticket

[Order Details](https://github.com/spooky-oysters/bangazon/issues/10)

## Description of Proposed Changes

- Requests for single Orders return all products associated with it

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout dr-ordersWithProducts
dotnet run
```
- GET http://localhost:5000/api/Order
- - Shows all orders
- GET http://localhost:5000/api/Order/{id}
- - Should return a single order with all products

## Impacted Areas in Application

* GET requests for single orders

## Mentions @username

Tag users that need to review this code
